src/stories/Select.stories.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react'
+import {Meta} from '@storybook/react'
+
+import {BaseStyles, Text, Select, ThemeProvider} from '..'
+import {SelectProps} from '../Select'
+
+export default {
+  title: 'Forms/Select',
+  component: Select,
+  decorators: [
+    Story => {
+      return (
+        <ThemeProvider>
+          <BaseStyles>
+            <Story />
+          </BaseStyles>
+        </ThemeProvider>
+      )
+    }
+  ],
+  argTypes: {
+    block: {
+      defaultValue: false,
+      control: {
+        type: 'boolean'
+      }
+    },
+    contrast: {
+      defaultValue: false,
+      control: {
+        type: 'boolean'
+      }
+    },
+    disabled: {
+      defaultValue: false,
+      control: {
+        type: 'boolean'
+      }
+    },
+    required: {
+      defaultValue: false,
+      control: {
+        type: 'boolean'
+      }
+    },
+    sx: {
+      table: {
+        disable: true
+      }
+    },
+    variant: {
+      name: 'Variant',
```

Comment:
can we call this `size`?

Reply:
Yes, absolutely. I was waiting for https://github.com/primer/react/pull/1661 to merge which deprecates `variant` and adds `size`.